.gitignore

```diff
@@ -0,0 +1,20 @@
+**/.DS_Store
```

Comment:
Please don't include the .gitignore file in the submission.

Reply:
Ouch ... sorry. It must have been accidentally pushed when I've rebased from leia to matrix.
Is there a reason why on matrix .gitignore has been removed?
It's a bit hard switching branches from leia to matrix because of missing .gitignore on matrix branch.

Reply:
@pkscout Done 👍

Reply:
As far as I know, we've always asked that the .gitignore file be removed, or at least for as long as I've been submitting stuff. I know I've had to be reminded of it myself from time to time. '-) Oddly, your Leia submission didn't include a .gitignore, so I was able to approve that. It's just waiting until this gets approved so that it'll pass the Travis tests.

Reply:
The Leia submission PR doesn't show gitignore because is part of the branch: https://github.com/xbmc/repo-plugins/blob/leia/.gitignore 
While the same file doesn't exist on matrix.
So, every time I switch branch ... I get that .gitignore from leia around in the unstaged files